content/docs/alerting/configuration.md

```diff
@@ -526,7 +526,21 @@ The fields are documented in the [Slack API documentation](https://api.slack.com
 type: <tmpl_string>
 text: <tmpl_string>
 url: <tmpl_string>
-[ style: <tmpl_string> [ default = '' ]
+[ name: <tmpl_string> ]
```

Comment:
This isn't completely true as either url or name are mandatory.

Also it would be useful to have a link to https://api.slack.com/docs/interactive-message-field-guide#action_fields.

Reply:
added link and note